src/django/api/extended_fields.py

```diff
@@ -19,16 +19,28 @@ def extract_int_range_value(value):
 MAX_PRODUCT_TYPE_COUNT = 50
 
 
-def get_facility_and_processing_type_extendfield_value(field,
-                                                       field_value, sector):
-    processed_values = field_value.get('processed_values')
-    raw_values = field_value.get('raw_values')
-
+def get_matched_values(field_value, sector):
     results = []
-
-    for value in processed_values:
+    for value in field_value:
         result = get_facility_and_processing_type(value, sector)
         results.append(result)
+    return results
+
+
+def get_facility_and_processing_type_extendfield_value(
+    field_value, sector,
+    isClaimEdit=False
```

Comment:
@VadimKovalenkoSNF Is there a case that it could be True? As I see the call of this function didn't changed so probably we just need to leave logic that is inside statement isClaimEdit equal False.

Reply:
@roman-stolar Yes, this is the case where it should be true: https://github.com/opensupplyhub/open-supply-hub/pull/214/files#diff-a1f72a03fbc15ab5b54ff60262b02d3adfda1735e3bfe12bf88e203798df4b76R195